src/snowflake/snowpark/file_operation.py

```diff
@@ -282,3 +284,66 @@ def put_stream(
         result_meta = cursor.description
         put_result = result_set_to_rows(result_data, result_meta)[0]
         return PutResult(**put_result.asDict())
+
+    def get_stream(
+        self,
+        stage_location,
+        *,
+        parallel: int = 10,
+        statement_params: Optional[Dict[str, str]] = None,
```

Comment:
I just found that put_stream doesn't have `statement_params`? Also should we move `decompress` ahead of `statement_params`?